lib/sly-completion.el

```diff
@@ -216,8 +216,14 @@ COMPLETIONS is a list of propertized strings."
            finally return (list formatted nil)))
 
 (defun sly-completion-annotation (completion)
-  "Grab the annotation of COMPLETION, a string, if any"
-  (get-text-property 0 'sly--annotation completion))
+  "Grab the classification of COMPLETION, if any, and generate a string for display."
```

Comment:
The previous version didn't have this `when-let` logic, but since you introduced it, I think it makes sense, so let's keep it.  

But it should do something if only one of the pieces of info are available.  So please use something similar to:
```
(string-join (delete nil `(,classification
                             ,(and score (format "%5.2f%%" (* score 100)))))
               " ")
```
`string-join` has existed since 24.4, so it's safe.

This will return the empty string if neither classification or score exist, but that's just fineI think it's fine.